nodes/nodes-containers-downward-api.adoc

```diff
@@ -0,0 +1,41 @@
+:context: nodes-containers-downward-api
+[id='nodes-containers-downward-api_{context}']
+= Allowing containers to consume API objects
```

Comment:
Is this really just one user story? You're creating so many different pod.yaml files.

Reply:
It all came from one topic originally.

Reply:
I think you should still consider breaking it into smaller stories. There were lots of old topics that went all over the place.

Reply:
Time pending. On the list.